### ISSUE TITLE
The procedure in F65 is too implementation-specific

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wcag#5272]
The procedure in [F65](https://www.w3.org/WAI/WCAG22/Techniques/failures/F65) appears to focus on specific implementation patterns rather than the expected accessibility outcome:

> Procedure
> Identify img, area, and input elements of type image. For each of these elements:
> 1. Check if the alt attribute is present.
> 2. Check if aria-labelledby attribute is present AND references one or more id attributes in the page AND check if aria-labelledby is [accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported).
> 3. Check if the aria-label attribute is present AND check if aria-label is [accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported).
> 4.Check if the title attribute is present AND check if title is [accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported).
>
> Expected Results
> If all of # 1, # 2, # 3 and # 4 are false then this failure condition applies.

- one issue is that input type="image" can receive its accessible name from an associated label. As a result, some scenarios that would be classified as failures by the current procedure may, in fact, conform to the SC;
- additionally, the procedure does not account for images that are intentionally removed from the accessibility tree. In those cases, the image is not exposed to assistive technologies, so the absence of an accessible name would not constitute a failure.

As with some of the other issues raised for failure techs, I recommend focusing on the expected outcome rather than the implementation details.
